private-undistributed.yml

```diff
@@ -90,7 +92,6 @@ allow-licenses:
   - 'MIT AND MPL-2.0'
   - 'MIT AND PSF-2.0'
   - 'MIT AND Python-2.0.1'
-  - 'MIT AND ZPL-2.1'
```

Comment:
Any reason why we would remove this one?

Reply:
No, that's a hilarious snafu on my part. Fixing pronto!